packages/babel-helper-builder-react-jsx/src/index.js

```diff
@@ -44,6 +46,12 @@ export default function(opts) {
         convertJSXIdentifier(node.object, node),
         convertJSXIdentifier(node.property, node),
       );
+    } else if (t.isJSXNamespacedName(node)) {
+      /**
+       * If there is flag "throwIfNamespace"
+       * print XMLNamespace like string literal
+       */
+      return t.stringLiteral(node.namespace.name + ":" + node.name.name);
```

Comment:
I'm not really sure about this, should I make some helper for it? (I've discovered that there is `getQualifiedJSXName`, so maybe something in this way.)

Reply:
Can a namespace be nested? I don't think a helper is necessary, and you can use a template literal here as well

Reply:
Ok looks like they can't be nested (parse error)

Reply:
I don't think that namespace could be nested. Ok, I will go for template literal, I've wanted to stay in style with `getQualifiedJSXName` implementation. 😆